signwriting_transcription/pose_to_signwriting/data/config.py

```diff
@@ -70,17 +70,17 @@ def create_config(data_path="/output/poses", experiment_dir='/model/poses'):
         adam_betas: [0.9, 0.98] 
         scheduling: "plateau"
         patience: 10
-        learning_rate: 0.0002
-        learning_rate_min: 0.00000001
+        learning_rate: 0.00015
+        learning_rate_min: 0.000000015
         weight_decay: 0.0
         label_smoothing: 0.1
         loss: "crossentropy-ctc"       # use CrossEntropyLoss + CTCLoss
         ctc_weight: 0.3                # ctc weight in interpolation
         batch_size: 4                  # much bigger than text! your "tokens" are "frames" now.
         batch_type: "sentence"
         batch_multiplier: 1
-        # early_stopping_metric:       # by default, early stopping uses "fsw_eval" metric
-        epochs: 15                     # Decrease for when playing around and checking of working.
+        early_stopping_metric: chrf        # by default, early stopping uses "fsw_eval" metric
```

Comment:
why chrf and not your signwriting evaluation method?

Reply:
The reason is mostly technical, the problem is that there is a data checking process of the joeynmt themselves that needs to be verified that the parameter belongs to a certain group of matrices that because we didn't overwrite the file (sounds a bit redundant to change a line in the code) we need to transfer a matrix that it knows. After that in any case we run him back to our truck of course. Just to get context where it is in the code:
<img width="514" alt="Screenshot 2024-06-22 at 11 14 16" src="https://github.com/sign-language-processing/signwriting-transcription/assets/112009232/c602cbd3-298d-4fd5-a7fc-8b5e33ac315f">